### ISSUE TITLE
BingTileGridGenerator very slow for country-wide generations at high resolutions

### ISSUE THREAD

**Issue by alronlam** [thinkingmachines/geowrangler#212]
For PH Bing Tile Zoom Level 16, it takes ~24 hours to generate as reported by @tm-dafrose-bajaro using commit https://github.com/thinkingmachines/geowrangler/commits/4b6292354c16b8676f868b7d69e505930730687b

Here's a minimal code snippet to replicate:

```
!wget https://media.githubusercontent.com/media/wmgeolab/geoBoundaries/a6ef6576d347e3885a3ec4891b95b26e76668cbe/releaseData/gbOpen/PHL/ADM0/geoBoundaries-PHL-ADM0-all.zip 
!unzip geoBoundaries-PHL-ADM0-all.zip 

import geopandas as gpd
from geowrangler.grids import BingTileGridGenerator

gdf = gpd.read_file("geoBoundaries-PHL-ADM0_simplified.geojson").set_crs("epsg:4326")
grids_gdf = BingTileGridGenerator(16).generate_grid(gdf)
```

Is there any way we can speed this up?

**Comment by joshuacortez** [comment]
Addressed by #245

**Comment by joshuacortez** [comment]
@alronlam see sample usage [here](https://geowrangler.thinkingmachin.es/tutorial.grids.html#fast-bing-tile-grid-generator)! The faster grid generator is merged to master already
